frontend/src/containers/Home/product-list/ProductItem.jsx

```diff
@@ -57,7 +57,7 @@ ProductItem.defaultProps = {
 };
 
 ProductItem.propTypes = {
-  item: PropTypes.objectOf(PropTypes.object)
+  item: PropTypes.objectOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string]))
```

Comment:
this should be string only. when we start taking the info from the backend we'll have strings as ids, not number. reason you're getting numbers also is probably due to the static file having different types for ids